blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
```

Comment:
I guess so, but "scratchblocks" is at least specific about what this is. If you google "scratchblocks" then your project at least comes up at the top so it's discoverable. 

Also, since this is only for the Scratch forum (at the moment) we might just say "export for scratch forum" (and update it as needed in the future)